Runtime/ButtonMethodAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace UnityAddressableImporter.Helper
+{
+    using System;
+
+    [AttributeUsage(AttributeTargets.Method)]
+    public class ButtonMethodAttribute : PropertyAttribute
```

Comment:
Why did this move to runtime asmdef?

Reply:
Move Under Editor asm.def into folder Attributes